reports/saksham.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: report
+permalink: /GSoC/2018/StatusReport/SakshamBansal
+student: Saksham Bansal
+organisation: coala
+organisation_link : https://coala.io
+project: Improve installation of coala and coala-bears
+project_link: https://summerofcode.withgoogle.com/projects/#4642118669172736
+tarball: https://rawgit.com/anctartica/GSoC-Report/master/work.tar.gz
+mentors: >
+ [Muhammad Kaisar Arkhan](https://github.com/yukiisbored) & [Matt Bodenhamer]
+ (https://github.com/mbodenhamer) & [Andrey Safonov](https://github.com/xferra)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/10
```

Comment:
see https://github.com/coala/projects/issues/627 and https://github.com/coala/projects/issues/615